frontend/src/main/scala/bloop/engine/Interpreter.scala

```diff
@@ -157,6 +161,38 @@ object Interpreter {
     else reportMissing(missing, state)
   }
 
+  private def run(cmd: Commands.Run, state: State): State = {
+    val reporter = ReporterConfig.getDefault(cmd.scalacstyle)
+    def getMainClass(state: State, project: Project): Option[String] = {
+      RunTasks.findMainClasses(state, project) match {
+        case Array() =>
+          state.logger.error(s"No main classes found in project '${project.name}'")
+          None
+        case Array(main) =>
+          Some(main)
+        case mainClasses =>
+          state.logger.error(s"Several main classes were found, specify which one:")
```

Comment:
Instead of doing this, can we prettify it and put it all in one string? I'd like to have colors here too.

Reply:
I think that it's easier to read if each main class is on its own line, but I can change it if you really think that it's better on the same line.

I agree about making the output of bloop more colorful. We should probably come up with some "color scheme" and re-use the same colours to mean the same things everywhere. It's probably easier to get something consistent if we take care of improving all outputs in one PR. I'll open an issue.

Reply:
> I think that it's easier to read if each main class is on its own line, but I can change it if you really think that it's better on the same line.
  
I think you mistunderstood me, what I meant is that it's better if we have it all in one string (even if it has newlines to show every entry). As in,putting everything in a triple quote string that afterwards makes it easier to change, rather than printing this and then doing foreach.

Reply:
Oh, sure 😄